azure/template.json

```diff
@@ -572,6 +598,32 @@
                 "worker-app-service-plan"
             ]
         },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "wkr-postgresql-server-firewall-rules",
+            "type": "Microsoft.Resources/deployments",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'),'postgresql-server-firewall-rules.json')]",
```

Comment:
take it we own the template json

Reply:
Yes we do. There are building blocks for all the Azure resources we're likely to use.